dist/profile/manifests/buildmaster.pp

```diff
@@ -239,24 +240,26 @@
     }
 
     # Define Casc directory through java opts to avoid conditional environment variable
-    $java_opts = "${java_opts} -Dcasc.jenkins.config=${jcasc_config_dir}"
+    $jcasc_java_opts = " -Dcasc.jenkins.config=${container_jenkins_home}/${jcasc_config_dir}"
```

Comment:
improvement for later:
provide an auto reload token as well, and either a file watcher or call the webhook on change

Reply:
https://github.com/jenkins-infra/jenkins-infra/pull/1817